lib/iex/lib/iex/autocomplete.ex

```diff
@@ -269,6 +289,14 @@ defmodule IEx.Autocomplete do
     end
   end
 
+  defp match_map_fields(map, hint) do
+    map
```

Comment:
I'm glad to change that because I like `for` comprehensions...but I'm curious about your statement that it's faster given my use of stream operations (so the map is only traversed once).  Why are comprehensions faster?  On my team others have expressed a preference for piping a collection through a bunch of `Stream` operations instead of using for comprehensions for this kind of situation, but if comprehensions are faster I'd like to understand that so I can explain it to my teammates :).